lab_6_pipeline/pipeline.py

```diff
@@ -44,37 +92,60 @@ def __init__(self, lemma: str = "", pos: str = "", tags: str = ""):
         """
         Initializes MorphologicalTokenDTO
         """
+        self.lemma = lemma
+        self.pos = pos
+        self.tags = tags
 
 
 class ConlluToken:
     """
     Representation of the CONLL-U Token
     """
 
-    def __init__(self, text: str):
+    def __init__(self, text: str, morphological_parameters: MorphologicalTokenDTO, position: int):
```

Comment:
please, do not modify methods declarations
if you want to set morphological parameters may be you have other ways to do it?

Reply:
inspect set_morphological_parameters method more closely

Reply:
Done. I've changed my constructor. The main modification is to convert attributes with default values. Now, the constructor contains only one protected parameter _text and other attributes have default value. Before this, I've created constructor with explicit declaration of parameters inside the constructor.